hack/csv-generator.go

```diff
@@ -27,6 +27,8 @@ import (
 
 	"github.com/blang/semver"
 	gyaml "github.com/ghodss/yaml"
+	"github.com/google/go-containerregistry/pkg/name"
```

Comment:
Does this package need a remote registry to be available? If so, I don't think it would be a good idea to rely on that, maybe instead, we can change our `defaults.go` files to use digests instead of tags

Reply:
Not the package per say, but yes, it is used to query the repos listed in `defaults.go` to translate their tag into digests.
I can definitely just replace the tags with digests in `defaults.go` if desired, that would simplify this PR by a lot!

Reply:
Yes, I think it would be a better idea